extensions/amp-cxense-player/0.1/amp-cxense-player.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CSS} from '../../../build/amp-cxense-player-0.1.css';
+import {getLengthNumeral, isLayoutSizeDefined} from '../../../src/layout';
+import {loadPromise} from '../../../src/event-helper';
+import {addParamsToUrl} from '../../../src/url';
+import {listenForOnce, postMessage} from '../../../src/iframe-helper';
+import {getDataParamsFromAttributes, removeElement} from '../../../src/dom';
+import {setStyles} from '../../../src/style';
+
+const cxDefaults = {
+  apiHost: 'https://api.widget.cx',
+  distEmbedHost: 'https://embed.widget.cx',
+  debugEmbedHost: 'https://stage-embed.widget.cx',
+  distEmbedApp: '/app/player/m4/dist/',
+  debugEmbedApp: '/app/player/m4/debug/',
+  poster: 'https://i.imgur.com/dptGg1l.png',
```

Comment:
@gkindel can you publish this Cxense logo on a embed.widget.cx, I no longer have access
https://i.imgur.com/dptGg1l.png

Thanks